planner/property/stats_info.go

```diff
@@ -24,7 +24,7 @@ type StatsInfo struct {
 	RowCount    float64
 	Cardinality []float64
 
-	HistColl statistics.HistColl
+	HistColl *statistics.HistColl
```

Comment:
So it may be nil now? We should check all the places that use it to avoid nil pointer reference.

Reply:
In this pr, it's safe. DataSource always holds a histogram.